src/common/TransactionParser.ts

```diff
@@ -68,9 +67,10 @@ export class TransactionParser {
         return newTransaction;
     }
 
-    private extractTransactionData(block: any, transaction: any) {
+    extractTransactionData(block: IBlock, transaction: ITransaction) {
         const from = String(transaction.from).toLowerCase();
-        const to = String(transaction.to).toLowerCase();
+        const to: string = transaction.to === null ? "" : String(transaction.to).toLowerCase();
+        const addresses: string[] = to ? [from, to] : [from];
```

Comment:
if `to` == null, then `contract` should exist and you need to save `contract` address

Reply:
Not at this point, later in the code.

Reply:
let's do it now, that's the whole point of this change